solidity/random-beacon/contracts/RandomBeaconGovernance.sol

```diff
@@ -484,10 +499,20 @@ contract RandomBeaconGovernance is Ownable {
         onlyAfterGovernanceDelay(groupCreationFrequencyChangeInitiated)
     {
         emit GroupCreationFrequencyUpdated(newGroupCreationFrequency);
+        (
+            ,
+            uint256 groupLifetime,
+            uint256 dkgResultChallengePeriodLength,
+            uint256 dkgResultSubmissionTimeout,
+            uint256 dkgSubmitterPrecedencePeriodLength
+        ) = randomBeacon.groupCreationParameters();
```

Comment:
Ah, this was the point of all of the refactorings in `RandomBeacon.sol`. Rather than call the contract a bunch of time for individual fields, we can call the fields in bundles

Reply:
Most importantly, this saves us bytecode space. Every public field and function occupies more bytecode than internal field or function. We are very close to the limit and we really need this space for `availableRewards` view function that should be added to Threshold `IApplication`.